src/openfermion/utils/_trotter_exp_to_qgates_test.py

```diff
@@ -12,8 +12,10 @@
 """Tests for _trotter_exp_to_qgates.py"""
 
 import unittest
-from _trotter_exp_to_qgates import *
-from _trotter_exp_to_qgates import _third_order_trotter_helper
+#from _trotter_exp_to_qgates import *
+#from _trotter_exp_to_qgates import _third_order_trotter_helper
```

Comment:
Remove these commented out lines :P

Reply:
I knowwww, you beat me to it, I was just quickly seeing if this is the reason python3 didn't work. Since python3 was working on my comp but not via Travis. And it looked like I was right! Removing now.